src/workspaces/ShareWorkspaceModal/ShareWorkspaceModal.ts

```diff
@@ -112,7 +113,8 @@ const ShareWorkspaceModal: React.FC<ShareWorkspaceModalProps> = (props: ShareWor
     const numAdditions = _.filter(({ email }) => !_.some({ email }, originalAcl), acl).length;
     const eventData = { numAdditions, ...extractWorkspaceDetails(workspace.workspace) };
 
-    const aclUpdates = [
+    // @ts-ignore
+    const aclUpdates: WorkspaceAclUpdate[] = [
```

Comment:
This flow logic seems to be too complicated for TypeScript (or me) to understand.